qiskit/backends/ibmq/credentials/__init__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+Utilities for working with credentials for the wrapper.
+"""
+from collections import OrderedDict
+import logging
+
+from qiskit import QISKitError
+from .credentials import Credentials
+from ._configrc import read_credentials_from_qiskitrc, store_credentials
+from ._environ import read_credentials_from_environ
+from ._qconfig import read_credentials_from_qconfig
+
+logger = logging.getLogger(__name__)
+
+
+def discover_credentials():
+    """
+    Automatically discover credentials for online providers.
+
+    This method looks for credentials in the following locations, in order,
+    and returning as soon as credentials are found::
+
+        1. in the `Qconfig.py` file in the current working directory.
+        2. in the environment variables.
+        3. in the `qiskitrc` configuration file.
```

Comment:
Is there a canonical location for the qiskitrc? I assume ~/.qiskitrc or something, we probably should put this here to be explicit where this function will look.

Reply:
Yes, it's referenced in other docstrings, but indeed would make sense to clarify it here :+1: